CNI/romana

```diff
@@ -154,9 +155,9 @@ set_up_pod () {
 	# Asking romana ipam for an IP address, based on `tenant` label.
 	log "--- PRE-IPAM ---"
 	[[ $SEGMENT ]] || SEGMENT=default
-	IP=$(ipam_allocate_ip $TENANT $SEGMENT $NODE)
+	IP=$(ipam_allocate_ip $NAMESPACE $SEGMENT $NODE)
```

Comment:
Actually i commented TENANT variable and use NAMESPACE everywhere. Maybe we will change it later.